### PR TITLE
Bump three dependencies

### PR DESCRIPTION
**Pull request by bjornstar**
- Bump three & @types/three to `r137`
- Make types more flexible so we don't have to match the entire Scene type, just the parts we need.
- Likewise with World
- ['.gitignore'] Add `yarn-error.log`

### CODE REVIEW

**Comment by marcofugaro** [comment]
Thanks!

> Make types more flexible so we don't have to match the entire Scene type, just the parts we need.

Could you explain the reason behind this? What's wrong from importing the Scene type from three?

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+yarn-error.log
```
Comment (marcofugaro) [inline .gitignore]:
```suggestion
*.log
```

**Comment by bjornstar** [comment]
> Thanks!
> 
> > Make types more flexible so we don't have to match the entire Scene type, just the parts we need.
> 
> Could you explain the reason behind this? What's wrong from importing the Scene type from three?

It reduces the footprint for potential conflicts. This library only needs to be able to add & remove meshes on a scene as well as iterate over bodies on a world. I ran into an issue where there was a type mismatch on Scene related to some fog property. By being selective in declaring what we need we can avoid these types of issues.

**Comment by marcofugaro** [comment]
> I ran into an issue where there was a type mismatch on Scene related to some fog property.

Can you show me what was the problem? cannon-es-debugger uses the `@types/three` library you install locally, not the one specified in this package.json. So ideally there shouldn't be any version mismatch.

**Comment by bjornstar** [comment]
> > I ran into an issue where there was a type mismatch on Scene related to some fog property.
> 
> Can you show me what was the problem? cannon-es-debugger uses the `@types/three` library you install locally, not the one specified in this package.json. So ideally there shouldn't be any version mismatch.

It's fine, I don't need these changes.

**Pull request closed without merging**